Update deps to fix runtime panic on run

nikolaiwarner opened the pull request:
fixes #2 ... Seems that an updated set of dependencies is enough to fix the panic. Yay!

nikolaiwarner committed:
Update cargo.lock

nikolaiwarner committed:
Update deps to fix runtime panic on run

dbaynard commented:
Thanks! (and thanks for your patience). I’ll do a quick review of dependencies in the next few days. This has been helpful.

dbaynard committed:
Bump version to 0.1.2

dbaynard committed:
Explicitly allow integer overflow in test

Overflow here is safe and preserves the invariant, and I now need to be
explicit to avoid a panic.

In release mode, there was never a panic, but the code didn’t express
that overflow is safe.

dbaynard commented:
Thanks @nikolaiwarner — do let me know if there are any features, etc that you’d like.
